### ISSUE TITLE
Code and Equation doesn't display correctly :(

### ISSUE THREAD

**Issue by min-hieu** [NotionX/react-notion-x#294]
#### Description

The equation is not syntax colored and the equation doesn't seem to be centered. I tried checking the official minimal example but it didn't work on there either. I think this is an important feature that should be fixed soon. I'm currently using version 6.12.9 for react-notion-x and notion-client. I used simple code similar to the minimal example.

#### Notion Test Page ID

<img width="542" alt="image" src="https://user-images.githubusercontent.com/53557912/169526564-1a77550c-4f16-43cc-b2d1-db4463b2b84c.png">

#### Official react-notion with equation and code block not working
https://react-notion-x-minimal-demo-saasify.vercel.app/0c322c33381c49bca5083a451c334c39#886d742ac1f743ddb033e0b837c005c9
and 
https://react-notion-x-minimal-demo-saasify.vercel.app/7820b2d5300747b38e31344eb06fbd57

**Comment by dmathewwws** [comment]
I believe this issue shows how to resolve that: https://github.com/NotionX/react-notion-x/issues/20

**Comment by min-hieu** [comment]
I don't think it does. This has nothing to do with the code styling or the styling of the math display.
